docs/MODULES.md

```diff
@@ -247,6 +254,40 @@ This module is used for decoding a QR in image (if present).
            .addSteps('decode-qr',options)
            .run()
 ```
+
+## dither-module
+
+[more info on wikipedia] : "wikipedia-link":"https://en.wikipedia.org/wiki/Dither"
```

Comment:
Can you please add the link at the end of the description text instead of adding at the beginning? Also please add the MD properly as shown in the comments below

Reply:
Yess, done with it.